tests/ffi/common/prepare.sh

```diff
@@ -37,6 +37,18 @@ reload_config() {
 }
 
 prepare_images() {
+   # Update container image_copy_tmp_dir if the image is an OStree.
+   # Default location for storing temporary container image content. Can be overridden with the TMPDIR environment
+   # variable. If you specify "storage", then the location of the
+   # container/storage tmp directory will be used.
+   # By default image_copy_tmp_dir="/var/tmp"
+   # This is a work around and it should not be used constantly
+   if [ -d /run/ostree ]; then
+      exec_cmd "mkdir -p /var/qm/tmp.dir"
+      exec_cmd "mkdir -p /etc/qm/containers/containers.conf.d"
+      exec_cmd "echo -e '[engine]\nimage_copy_tmp_dir=\"/var/tmp.dir\"' > /etc/qm/containers/containers.conf.d/qm_image_tmp_dir.conf"
```

Comment:
I would add here check id the file exist, and a value exist, WDYT?
What do you prefer to to if each of the following  exist? 

I assume that this step will done only once on the first prepare, after that, id the file exist and the parameter, just verify it is the correct one,

I would also check this directory in cleanup,
If exist, remove the files there

Reply:
I can add a checkup for the directory that I'm creating beforehand and if exists, then I will need to create another random empty directory in /var/qm/ and in /etc/qm/containers/ on host.
This is a workaround, so I don't think that I should enhance this patch too much, as the image should be fixed, rather than this patch should be extended too much.
Also an image comes clean, without these directories, so chance of have the same directory existing from clean image is low.
I ran several times this code on the system with existing directory and no harm, all worked just fine, so I doubt that this enhancement will be beneficial.

Reply:
OK, no need, you are correct